src/backend.js

```diff
@@ -24,8 +24,10 @@ $axios.interceptors.response.use(function (response) {
 export default {
   voteCast (isRight, imageLeft, imageRight, timeTaken, textComment) {
     let sessionHash = localStorage.getItem('streetwiseSession') || null
+    let campaignId = localStorage.getItem('currentCampaignId') || null
```

Comment:
[Minor]: Prefer `const` over `let` for vars that are not changed in the scope.

Reply:
Sure, please feel free to just fix this kind of thing.